### PR TITLE
Update RAF Documentation

### PR DESCRIPTION
**Pull request by mendelB**
### What's this PR do?

This pull request updates the Refer a Friend documentation per the V2 changes

### How should this be reviewed?
👀 

### Relevant tickets

References [Pivotal #173589730](https://www.pivotaltracker.com/story/show/173589730).

### Checklist

- [x] This PR has been added to the relevant Pivotal card.
- [x] Documentation added for new features/changed endpoints.

### CODE REVIEW

File: docs/development/features/referral-pages.md [hunk 1]
```diff
@@ -33,7 +35,21 @@ The user's Alpha Referral Page prompts the user to share their Beta Referral Pag
...
-If the `campaign_id` query parameter is missing completely, an `ErrorPage` component will displayed.
+If the `campaign_id` query parameter is missing completely, the campaign will default to the assigned `DEFAULT_REFERRAL_CAMPAIGN_ID` variable for the application. (If this variable is not assigned, the campaign link will display an error).
+
+### Account Alpha Referral Page
+
+```
+dosomething/us/account/refer-friends
+```
+
+User's can visit their account RAF tab for a similar experience to the cannonical Alpha Referral Page.
```
Comment (aaronschachter) [inline docs/development/features/referral-pages.md:46]: Nit - Should be `Users`, and typo in `canonical` 🏴‍☠️

File: docs/development/features/referral-pages.md [hunk 2]
```diff
@@ -45,9 +61,13 @@ The Beta Referral Page links to the campaign that the beta should join (set via
...
 
-If the `campaign_id` query parameter is missing completely, an `ErrorPage` component will displayed.
+We also display a secondary campaign link using the configured `DEFAULT_REFERRAL_CAMPAIGN_ID`.
+
+![Secondary Campaign Link Example](../../.gitbook/assets/secondary-referral-campaign.png)
 
-The campaign URL that the Beta Page links to will include the alpha's user ID as a `referrer_user_id` query parameter:
+If the `campaign_id` query parameter matched this assigned default, we'll only display the primary link. If the `campaign_id` query parameter is missing completely, we'll similarly pull the default ID and display it _as_ the primary link.
+
+The campaign URL/s that the Beta Page links to will include the alpha's user ID as a `referrer_user_id` query parameter:
```
Comment (aaronschachter) [inline docs/development/features/referral-pages.md:70]: Typo `/`

File: docs/development/features/referral-pages.md [hunk 3]
```diff
@@ -2,16 +2,18 @@
...
-The Refer A Friend Feature provides members with a unique URL to share with their friends, to get them to register and signup for a campaign on DoSomething.org.
+The Refer A Friend (RAF) Feature provides members with a unique URL to share with their friends, to get them to register and signup for a campaign on DoSomething.org.
 
 We commonly use these terms (in the business, and in code) when describing this feature:
 
 - **Alpha** - The user who is sharing the URL with friends, prompting the friends to register
 - **Beta** - The friends that the alpha is sharing the URL with
 
-Both the alpha and beta may earn a reward for the beta registering and signing up for the campaign.
+Both the alpha and beta _may_ earn a reward for the beta registering and signing up for the campaign.
```
Comment (aaronschachter) [inline docs/development/features/referral-pages.md:12]: This feels a little confusing - what do you think about scrapping this sentence completely, and adding context to line 14, e.g. `Previous iterations of this feature did offer rewards, see History`

File: docs/development/features/referral-pages.md [hunk 4]
```diff
@@ -33,7 +35,21 @@ The user's Alpha Referral Page prompts the user to share their Beta Referral Pag
 
 ![Alpha Referral Page Example](../../.gitbook/assets/alpha-referral-page.png)
 
-If the `campaign_id` query parameter is missing completely, an `ErrorPage` component will displayed.
+If the `campaign_id` query parameter is missing completely, the campaign will default to the assigned `DEFAULT_REFERRAL_CAMPAIGN_ID` variable for the application. (If this variable is not assigned, the campaign link will display an error).
+
+### Account Alpha Referral Page
```
Comment (aaronschachter) [inline docs/development/features/referral-pages.md:40]: What do you think about changing the "Alpha Referral Page" header to read as "Campaign Alpha Referral Page" to better differentiate the two?

**Comment by mendelB** [comment]
> Just a few suggestions and nitpicks, I think I was a grammar teacher in a former life 👻 📚

Lol they never taught us grammar in Rabbinical school so this is helpful! 🙌

**Pull request merged**